.github/workflows/qodana_code_quality.yml

```
@@ -0,0 +1,20 @@
+
+jobs:
+  qodana:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: 'Qodana Scan'
+        uses: JetBrains/qodana-action@v2023.3
+        env:
+          QODANA_TOKEN: ${{ secrets.QODANA_TOKEN }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)